### PR TITLE
Soolaf_homework Week1

### PR DESCRIPTION
**Pull request by soolaf**

### CODE REVIEW

File: Week1/homework/db.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  var result = "SELECT * FROM countries WHERE countries.country_name LIKE '%land%'";
+  connection.query(result, function(error, result) {
+    if (error) throw error;
+    console.log('countries with the word land\n', result);
+  });
+}
+
+function populationInBetween() {
+  var result =
+    'SELECT country_name, country_population FROM countries WHERE countries.country_population BETWEEN 500000 AND 1000000';
```
Comment (zhaobn) [inline Week1/homework/db.js]: the `countries.` part is not necessary in this case

File: Week1/homework/Create-table.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+];
+
+for (var i in create_tables) {
+  connection.query(create_tables[i], function(error, results) {
+    if (error) throw error;
+    console.log('Created new table');
+  });
+}
+
+var countries = [
```
Comment (zhaobn) [inline Week1/homework/Create-table.js]: You can do this, or take the data part out and make use of this helper https://github.com/mysqljs/mysql#escaping-query-identifiers

File: Week1/homework/db.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  connection.query(result, function(error, result) {
+    if (error) throw error;
+    else if (result == 0) console.log('No countries with population between 500,000 and 1 million');
+    else console.log(result);
+  });
+}
+
+function euroCountries() {
+  var result =
+    "SELECT countries.country_name, countries.continent FROM countries WHERE countries.continent LIKE '%Europe%'";
```
Comment (zhaobn) [inline Week1/homework/db.js]: To be strict, `= 'Europe'` is fine

File: Week1/homework/db.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  var result = 'SELECT * FROM countries ORDER BY countries.surface_areas DESC';
+  connection.query(result, function(error, result) {
+    if (error) throw error;
+    console.log('countries in descending order\n', result);
+  });
+}
+
+function rotterdamPopulation() {
+  var result =
+    "SELECT cities.city_population AS Rotterdam_Population FROM cities WHERE cities.city_name ='Rotterdam'";
```
Comment (zhaobn) [inline Week1/homework/db.js]: Same thing, you don't have to use `cities.` as long as everything is in the same table

File: Week1/homework/db.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  });
+}
+
+
+function topBySurface() {
+  var result =
+    'SELECT country_name,surface_areas FROM countries ORDER BY countries.surface_areas ASC LIMIT 10';
+  connection.query(result, function(error, result) {
+    if (error) throw error;
+    console.log('top 5 countries by Surface Area ', result);
```
Comment (zhaobn) [inline Week1/homework/db.js]: Log your results and see whether there are top 5 or bottom 5?

File: Week1/homework/db.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  var result = 'SELECT * FROM cities GROUP BY city_population ORDER BY city_population DESC';
+  connection.query(result, function(error, result) {
+    if (error) throw error;
+    console.log('most populated cities', result);
+  });
+}
+
+function worldPopulation() {
+  var result =
+    'SELECT SUM(city_population) AS cities_population, SUM(country_population) AS world_population FROM cities INNER JOIN countries ON city_population + country_population';
```
Comment (zhaobn) [inline Week1/homework/db.js]: `+`?

Comment (soolaf) [reply Week1/homework/db.js]: @zhaobn   Thank you for your feedback

**Pull request closed without merging**